synthtool/gcp/gapic_bazel.py

```diff
@@ -65,34 +69,32 @@ def _generate_code(
         language: str,
         *,
         private: bool = False,
+        discogapic: bool = False,
         proto_path: Union[str, Path] = None,
         output_dir: Union[str, Path] = None,
         bazel_target: str = None,
     ):
         # Determine which googleapis repo to use
-        if not private:
-            googleapis = self._clone_googleapis()
+        source_repo = None
+        source_repo_name = None
+        if discogapic:
+            source_repo = self._clone_discovery_artifact_manager()
```

Comment:
"source" is still kinda ambiguous.  Could rename these variables:
source_repo_with_protos
name_of_source_repo_with_protos

Reply:
I'm happy to rename it, but `_proto` suffix would not work for the `discovery-artifact-manager` repo, because it is a repo with discovery docs (jsons). Maybe something like `api_definitions_repo`?

Reply:
I like `api_definitions_repo`.